src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigStorageLocation`1.cs

```diff
@@ -2,19 +2,40 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
 
 namespace Microsoft.CodeAnalysis.Options
 {
     /// <summary>
     /// Specifies that an option should be read from an .editorconfig file.
     /// </summary>
-    internal sealed class EditorConfigStorageLocation<T> : OptionStorageLocation, IEditorConfigStorageLocation
+    internal sealed class EditorConfigStorageLocation<T> : OptionStorageLocation, IEditorConfigStorageLocation2
```

Comment:
can't remember.  did you open a bug about wanting to update Naming-Config to support this as well?

Reply:
Yes, [https://github.com/dotnet/roslyn/issues/30086](https://github.com/dotnet/roslyn/issues/30086)

---
In reply to: [219611551](https://github.com/dotnet/roslyn/pull/30043#discussion_r219611551) [](ancestors = 219611551)